tox.ini

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
+ignore = E203, E266, E501, W503
```

Comment:
When this was all first set up, Black did not support configuration in _pyproject.toml_, and obviously Flake8 still doesn't. At the time, Black recommended _tox.ini_ instead of _setup.cfg_ (and still did until very recently). So that is why — for items that we couldn't put into _pyproject.toml_ —  we ended up putting everything else into _tox.ini_.

That was then; this is now. Obviously the best solution would be for Flake8 to support _pyproject.toml_, but until that day arrives, I have no problem moving Flake8 configuration into _setup.cfg_ if that's preferred.

As for the Flake8 rules, E203 is ignored for Black compatibility purposes: https://black.readthedocs.io/en/stable/compatible_configs.html#id1

I'm not sure I agree that it's all invalidated by Black, which has opinions about some things but not others.

While I can't recall why both max-line-length and E501 are specified, it's quite possible that specifying both doesn't make any sense. The easy solution is to get rid of `max-line-length` here; if we instead remove E501, we'll have to decide whether we have an opinion on docstring line length (which Black does not have an opinion on).

I'm open to suggestions regarding to handle all these rules. Please let me know your thoughts.

Reply:
Here's a related experimental change in another plugin repo in which I removed both the E501 and E266 rules from the ignored list, adjusting two docstring lines to fit within the 88-character limit: https://github.com/pelican-plugins/similar-posts/commit/51d842994ce0bdecbbcc481a5d556f5da138ce9c

Reply:
Fixed in  https://github.com/pelican-plugins/image-process/pull/38/commits/b4cb65161dca3106745ef1e893e0a5bad74daef5